python_examples/sinric/_sinricsocket.py

```diff
@@ -1,37 +1,45 @@
-import websocket
-from base64 import b64encode as enc
 import json
-from ._mainqueue import queue
+from base64 import b64encode as enc
+
+import websocket
+from loguru import logger
+
 from ._cbhandler import CallBackHandler
+from ._mainqueue import queue
+
+logger.add("{}.log".format(__name__), rotation="10 MB")
 
 
 class SinricSocket:
     def __init__(self, apikey, callbacks):
-        self.apikey = apikey.encode('ascii')
+        self.apikey = apikey.encode("ascii")
         self.callbacks = callbacks
+        self.logger = logger
         self.callback_handler = CallBackHandler(self.callbacks)
```

Comment:
pass self.logger as parameter next to self.callbacks. So you can use self.logger in _cbhandler.py